server/routerlicious/packages/services-core/src/messages.ts

```diff
@@ -81,6 +82,9 @@ export interface IUpdateReferenceSequenceNumberMessage extends IObjectMessage {
  * Raw message inserted into the event hub queue
  */
 export interface IRawOperationMessage extends IObjectMessage {
+    // The type of the message
+    type: "RawOperation";
```

Comment:
you can use `typeof RawOperationType` so you don't need to repeat the string

Reply:
Thanks!